src/torchcodec/decoders/_core/VideoDecoder.h

```diff
@@ -288,9 +274,11 @@ class VideoDecoder {
   DecodeStats getDecodeStats() const;
   void resetDecodeStats();
 
-  double getPtsSecondsForFrame(int streamIndex, int64_t frameIndex);
-
  private:
+  explicit VideoDecoder(const std::string& videoFilePath, SeekMode seekMode);
```

Comment:
The constructors have to remain public. Yes, `initializeDecoder()` is private, but the constructors call that. I'd like to eventually remove the static functions `createFromFilePath()` and `createFromBuffer()` and just have all callers use the constructors.